cdc/server.go

```diff
@@ -257,27 +257,29 @@ func (s *Server) Close() {
 	}
 }
 
-func (s *Server) initDataDir(ctx context.Context) error {
-	if err := s.setUpDataDir(ctx); err != nil {
+func (s *Server) initDir(ctx context.Context) error {
+	if err := s.setUpDir(ctx); err != nil {
 		return errors.Trace(err)
 	}
 	conf := config.GetGlobalServerConfig()
-	err := os.MkdirAll(conf.DataDir, 0o700)
+	// Ensure data dir exists and read-writable.
+	diskInfo, err := checkDir(conf.DataDir)
 	if err != nil {
 		return errors.Trace(err)
 	}
-	diskInfo, err := util.GetDiskInfo(conf.DataDir)
+	log.Info(fmt.Sprintf("%s is set as data-dir (%dGB available), sort-dir=%s. "+
+		"It is recommended that the disk for data-dir at least have %dGB available space",
+		conf.DataDir, diskInfo.Avail, conf.Sorter.SortDir, dataDirThreshold))
+
+	// Ensure sorter dir exists and read-writable.
+	_, err = checkDir(conf.Sorter.SortDir)
```

Comment:
This logic and `checkDir` function are not tested now

Reply:
This lint is tested in integration tests, without it, many tests fails.

`checkDir` is covered by unit tests now.